dockerfiles/ClipperDevDockerfile

```diff
@@ -20,7 +20,7 @@ RUN pip install --upgrade pip
 
 RUN pip install -q cloudpickle==0.5.* pyzmq==17.0.* requests==2.20.0 subprocess32==3.2.* scikit-learn==0.19.* \
   numpy==1.14.* pyyaml>=4.2b1 docker==3.1.* kubernetes==6.0.* tensorflow==1.6.* mxnet==1.4.* pyspark==2.3.* \
-  xgboost==0.7.* jsonschema==2.6.* psutil==5.4.* prometheus_client
+  xgboost==0.7.* jsonschema==2.6.* psutil==5.4.* prometheus_client keras
```

Comment:
How about set Keras' version like this, `keras==2.2.*`? See https://pypi.org/project/Keras/#history.

Reply:
I agree. I will fix it. Thanks for the feedback!